### PR TITLE
Session recorder and hybrid performance

### PR DESCRIPTION
**Pull request by ppcano**
Rewrite and restructure the [current `Recording a session`](https://k6.io/docs/test-authoring/recording-a-session/) documentation to suggest the Hybrid approach. Updates:

[Create tests from recordings](https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/test-authoring/create-tests-from-recordings/)
- [Using the browser recorder](https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/test-authoring/create-tests-from-recordings/using-the-browser-recorder/)
- [Using the HAR converter](https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/test-authoring/create-tests-from-recordings/using-the-har-converter/)

### CODE REVIEW

**Comment by mdcruz** [comment]
@ppcano 

1. Not part of this change but worth adding a link to the load-testing types in this paragraph.

```
Edit your script as necessary. Depending on the type of testing, you might need to change different aspects of the script. Typical changes are for [load options](https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/using-k6/options) and to handle [correlation and dynamic data](https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/examples/correlation-and-dynamic-data).
```

type of testing -> add a link to https://k6.io/docs/test-types/load-test-types/

2. There is a broken link on [Using the browser recorder](https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/test-authoring/create-tests-from-recordings/using-the-browser-recorder/#trouble-try-the-har-converter) guide. The HAR converter link is pointing to this 404 page -> https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/test-authoring/recording-a-session/har-converter/

3. There is a broken link on [Using the HAR converter](https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/test-authoring/create-tests-from-recordings/using-the-har-converter/). The Browser recorder link is pointing to a 404 page -> https://mdr-ci.staging.k6.io/docs/refs/pull/1177/merge/test-authoring/recording-a-session/browser-recorder/

**Comment by ppcano** [comment]
@mdcruz the internal links are now fixed by https://github.com/grafana/k6-docs/pull/1177/commits/5d9e09412588101240fffbf72ab6ef9b6cf586a1

File: src/data/markdown/translated-guides/en/05 Test authoring/02 Create tests from recordings.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: 'Create tests from recordings'
+excerpt: 'In load testing, recording usually refers to the process of creating a load test from the recording of a user session.'
+---
+
+A recording stores the sequence of requests and parameters of a user session or API interactions. 
+You can then use this recording to auto-generate the logic of your test.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/05 Test authoring/02 Create tests from recordings.md:7]:
```suggestion
A recording stores the sequence of requests and parameters of a user session or API interaction. 
You can use this recording to auto-generate your test logic.
```

File: src/data/markdown/translated-guides/en/05 Test authoring/02 Create tests from recordings/01 Using the browser recorder.md [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
-The recorder captures every HTTP(s) request loaded into the browser as you click.
-This includes including ads, images, documents, etc.
 
 ## How to record
 
+>  **Note: the recorder is free to use.**
+>
+> The browser recorder requires having a [k6 Cloud account](https://app.k6.io), but you do not need an active k6 Cloud subscription.
+>
+> When you finish recording the session, the browser extension uploads the auto-generated k6 test into your k6 Cloud account. You can then copy the script to edit it in your local IDE and run the test locally using the `k6 run` command.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/05 Test authoring/02 Create tests from recordings/01 Using the browser recorder.md:24]:
```suggestion
<Blockquote mod="note" title="The recorder is free to use">

The browser recorder requires a [k6 Cloud account](https://app.k6.io), but you do not need an active k6 Cloud subscription.

When you finish recording the session, the browser extension uploads the auto-generated k6 test into your k6 Cloud account. You can then copy the script to edit it in your local IDE and run the test locally using the `k6 run` command. 

</Blockquote>
```

**Review by MattDodsonEnglish — APPROVED** [review]
LGTM. Made small nits

**Review by MattDodsonEnglish — APPROVED** [review]
LGTM. Made small nits

**Pull request merged**